toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/SceneView.kt

```diff
@@ -188,6 +197,11 @@ private fun SceneViewEventHandler(
     val currentOnPan by rememberUpdatedState(onPan)
 
     LaunchedEffect(Unit) {
+        launch {
+            sceneView.timeExtent.collect {
```

Comment:
can we follow mapView implementation here,
```suggestion
            sceneView.timeExtent.collect { currentTimeExtent ->
                currentTimeExtentChanged?.invoke(currentTimeExtent)
            }
```